gubernator/view_logs.py

```diff
@@ -167,6 +168,48 @@ def get_logs(build_dir, log_files, pod_name, filters, objref_dict):
     return all_logs, results, objref_dict, log_files
 
 
+def get_woven_logs(log_files, pod, filters, objref_dict):
+    logs = []
+    lines = []
+    combined_lines = []
+    first_combined = ""
+    pod_re = regex.wordRE(pod)
+
+    for log_file in log_files:
+        log = gcs_async.read(log_file).get_result()
+        logs.append(log.decode('utf8', 'replace'))
+    for log in logs:
+        lines.extend(log.split('\n'))
```

Comment:
I guess that information could be used to somehow differentiate the lines when they're woven together? For example the text could be in different colors for different logs... I don't know how useful that would be.